apps/docs/pages/guides/getting-started/quickstarts/kotlin.mdx

```diff
@@ -0,0 +1,190 @@
+
+    </StepHikeCompact.Details>
+
+    <StepHikeCompact.Code>
+
+     ```sql SQL_EDITOR
+      -- Create products table
+      create table
+        public.products (
+          _id bigint generated by default as identity not null,
+          productid text not null,
+          name text null,
+          description text null,
+          price real null,
+          image text null,
+          category text null,
+          nutrition text null,
+          constraint products_pkey primary key (productid),
+        ) tablespace pg_default;
+
+      ````
+
+    </StepHikeCompact.Code>
+
+  </StepHikeCompact.Step>
+
+  <StepHikeCompact.Step step={2}>
+
+    <StepHikeCompact.Details title="Create an Android app with Android Studio">
+      Setup environment for Android Development following official document.
+      Open Android Studio > New > New Android Project
+
+    </StepHikeCompact.Details>
+
+  </StepHikeCompact.Step>
+
+  <StepHikeCompact.Step step={3}>
+
+    <StepHikeCompact.Details title="Install the Supabase client library">
+    Setup Supabase libraries requires some others libraries from ktor, so make sure all dependencies are added.
+    Replace place holder of the dependency versions `$supabase_version` and `$ktor_version` with latest versions as below
+    <div style={{ display: 'flex', flexDirection: 'row', alignItems: 'center' }}> 
+      <a href="https://github.com/supabase-community/supabase-kt/releases" style={{ marginRight: '8px' }}> 
+          <img src="https://img.shields.io/github/release/supabase-community/supabase-kt?label=supabase-kt stable" alt="latest stable supabase-kt version"/> 
+      </a> 
+      <a href="https://github.com/supabase-community/supabase-kt/releases"> 
+          <img src="https://img.shields.io/github/v/release/ktorio/ktor.svg?label=ktor" alt="latest ktor version"/> 
+      </a> 
+    </div> 
+
+    </StepHikeCompact.Details>
+
+    <StepHikeCompact.Code>
+      ```kotlin
+        implementation "io.github.jan-tennert.supabase:postgrest-kt-android:$supabase_version"
+        implementation "io.ktor:ktor-client-android:$ktor_version"
+        implementation "io.ktor:ktor-client-logging-jvm:$ktor_version"
+        implementation "io.ktor:ktor-client-core:$ktor_version"
+        implementation "io.ktor:ktor-client-cio:$ktor_version"
+        implementation "io.ktor:ktor-utils-jvm:$ktor_version"
+      ```
+
+    </StepHikeCompact.Code>
+
+  </StepHikeCompact.Step>
+
+  <StepHikeCompact.Step step={4}>
+
+    <StepHikeCompact.Details title="Install the serializable plugin">
+    Open `build.gradle` (app), add serialization plugin to use annotation for data parsing
+
+    </StepHikeCompact.Details>
+
+    <StepHikeCompact.Code>
+      ```kotlin
+        plugins {
+            ...
+            id 'org.jetbrains.kotlin.plugin.serialization' version '1.7.0'
```

Comment:
Maybe replace `1.7.0` with a placeholder like `KOTLIN_VERSION` as the plugin version should always be the same as the kotlin version